src/pages/Bridge/helpers.ts

```diff
@@ -166,6 +166,7 @@ export const formatPoolValue = (amount: PoolBridgeValue) => {
   123456789012.123456 => 123.457B
   1234567890123.123456 => 1.23457T
  */
+// todo: deprecated, use formatDisplayNumber instead
```

Comment:
TODO

Reply:
I add these todo notes to remind guys later to use newly `formatDisplayNumber` instead of using these format function. I can't refactor all of these in one PR, it would taking so much time and the PR would be very big.